motors/tacho_motor_class.c

```diff
@@ -1198,9 +1196,29 @@ void tacho_motor_notify_position_ramp_down(struct tacho_motor_device *tm)
 }
 EXPORT_SYMBOL_GPL(tacho_motor_notify_position_ramp_down);
 
+int tacho_motor_get_state(struct tacho_motor_device *tm)
```

Comment:
Also need to move this function somewhere before `state_show()` so you don't have to declare the prototype.